src/lazyload.setSources.js

```diff
@@ -72,7 +72,11 @@ export const setSourcesBgImage = (element, settings) => {
 	}
 
 	if (bgDataValue) {
-		element.style.backgroundImage = bgDataValue;
+		element.style.backgroundImage = bgDataValue[0] === "u" ?
```

Comment:
This wouldn't work if the first applied background is a gradient. E.g. if `bgDataValue` is `linear-gradient(direction, color-stop1, color-stop2, ...), lazyImage.jpg`.